struct_amd64.go

```diff
@@ -262,3 +262,29 @@ func placeStack(v reflect.Value, addStack func(uintptr)) {
 func placeRegisters(v reflect.Value, addFloat func(uintptr), addInt func(uintptr)) {
 	panic("purego: not needed on amd64")
 }
+
+// shouldBundleStackArgs always returns false on non-Darwin platforms
+// since C-style stack argument bundling is only needed on Darwin ARM64.
+func shouldBundleStackArgs(v reflect.Value, numInts, numFloats int) bool {
+	return false
+}
+
+// structFitsInRegisters is not used on non-Darwin platforms.
+// This stub exists for compilation but should never be called.
+func structFitsInRegisters(val reflect.Value, tempNumInts, tempNumFloats int) (bool, int, int) {
+	panic("purego: structFitsInRegisters should not be called on non-Darwin platforms")
```

Comment:
The error message seems odd. Isn't this function defined for both Darwin and non-Darwin?

Reply:
True, it's just not needed on amd64 at all